backend/pages/utils/charts.py

```
@@ -348,4 +352,24 @@ def get_sankey_chart(df, chart_meta, csv_sheet_name, filter):
             'available': available_column_filters.tolist(),
             'selected': selected_column_filter
         }
-    }
\ No newline at end of file
+    }
+
+def get_table_chart(df, chart_meta, csv_sheet_name, filter):
+    # Convert DataFrame to list of lists for data
+    data = df.values.tolist()
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)